account/src/repositories/accountRepository.js

```diff
@@ -1,37 +1,38 @@
 
 import { MongoClient } from 'mongodb';
 
-async function getUsersCollection() {
-  const connectionURL = 'mongodb://jessica:123456789@mongodb:27017';
-  const connection = new MongoClient(connectionURL);
-  await connection.connect();
+console.log(process.env.DATABASE_URL);
```

Comment:
do you need this console.log?

Reply:
I used it just to do a test, but I redid the connection in main. I uploaded the change in the sprint 9.